tools/accuracy_checker/accuracy_checker/postprocessor/README.md

```diff
@@ -28,11 +28,13 @@ Accuracy Checker supports following set of postprocessors:
    - the preprocessing steps contains only one operation changing input image size, and the operation is `resize`
    - the preprocessing steps contains only two operations changing input image size, and the operations are `resize` and then `padding`.
    * `rescale` if this option enabled, rescaling boxes on input size will be performed before applying postprocessing (Optional, default `False`).
+* `letterbox_postprocessing_resize` - resizing normalized detection prediction boxes according to image with the letterbox resizing. Supported representations: `DetectionAnotation`, `DetectionPrediction`. Works only when the preprocessing steps contains only two operations changing - `resize` and then `padding` with `center` padding type.
 * `nms` - non-maximum suppression. Supported representations: `DetectionAnotation`, `DetectionPrediction`, `ActionDetectionAnnotation`, `ActionDetectionPrediction`.
   * `overlap` - overlap threshold for merging detections.
   * `use_min_area` - boolean value to determine whether to use minimum area of two bounding boxes as base area to calculate overlap.
-* `diou_nms` - distance-IoU non-maximum suppression. Supported representations: `DetectionAnotation`, `DetectionPrediction`, `ActionDetectionAnnotation`, `ActionDetectionPrediction`.
-  * `overlap` - overlap threshold for merging detections.
+  * `per_class_nms` - enables supression only within class (`False` by default)
+  * `diou` - uses distance-IoU non-maximum supression (`False` by default)
```

Comment:
I think, not all users know what is distance-IoU, please add link on resoulrse where it explained (e.g. https://arxiv.org/pdf/1911.08287v1.pdf)